### PR TITLE
Updated readme with consolidated dependencies

### PR DESCRIPTION
**Pull request by roberts01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,53 +68,48 @@ The idea is to install "Legacy Version" of WhatsApp on you device via adb and us
...
+```python
+pkg update && pkg upgrade
+pkg install python git
 ```
-2. Get python requirements
+
+2. Clone the repo
 ```python
-pip install -r requirements.txt
+git clone https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor.git
```
Comment (YuvrajRaghuvanshiS) [inline README.md:100]: && cd WhatsApp-Key-Databse-Extractor to enter into directory.

File: README.md [hunk 2]
```diff
@@ -68,53 +68,48 @@ The idea is to install "Legacy Version" of WhatsApp on you device via adb and us
...
+1. Get dependencies I
+```python
+pkg update && pkg upgrade
+pkg install python git
 ```
-2. Get python requirements
+
+2. Clone the repo
 ```python
-pip install -r requirements.txt
```
Comment (YuvrajRaghuvanshiS) [inline README.md:108]: These python requirements are different form termux requirements, must be included.

**Review by YuvrajRaghuvanshiS — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (roberts01) [reply README.md:100]: Happens in the termux_dependencies.py

Doesn't work if you try it with the && , tried it.

File: README.md (hunk 2 above)
Comment (roberts01) [reply README.md:108]: Included them in termux_dependencies.py

I added termcolur 
packaging doesn't exist (anymore?)
wget was already there

**Comment by roberts01** [comment]
Resolved all requests

File: README.md (hunk 2 above)
Comment (YuvrajRaghuvanshiS) [reply README.md:108]: > Included them in termux_dependencies.py

Is that on different branch? Can't see it here.

Comment (roberts01) [reply README.md:108]: https://github.com/YuvrajRaghuvanshiS/WhatsApp-Key-Database-Extractor/pull/28 its here

Comment (YuvrajRaghuvanshiS) [reply README.md:108]: termcolor is python module and can not be installed with pkg install. Don't merge python and shell requirements together.

Comment (roberts01) [reply README.md:108]: corrected it over in #28

File: README.md [hunk 3]
```diff
@@ -68,53 +68,57 @@ The idea is to install "Legacy Version" of WhatsApp on you device via adb and us
...
-### Installation (Not working?)
+### Installation and Usage 
+Note: run all of the following commands in Termux
 
-0. Connect adb to your phone. Run the following command in the Termux window: `adb connect localhost`
-
-1. Clone the repo
+1. Get dependencies I
+```python
+pkg update && pkg upgrade
```
Comment (YuvrajRaghuvanshiS) [inline README.md:94]: Quick fix. I'm seeing emoji here.

**Review by YuvrajRaghuvanshiS — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (roberts01) [reply README.md:94]: emoji?

**Pull request merged**